### PR TITLE
integration test: add agent LogIsQuiet test

### PR DESCRIPTION
**Pull request by dougsland**
Adding test for the LogIsQuiet option in the configuration. Related: https://github.com/containers/bluechi/issues/416

Signed-off-by: Douglas Schilling Landgraf <dougsland@redhat.com>

### CODE REVIEW

File: tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+    _, output = nodes[NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET} bluechi-agent should "
+                "NOT failed during the start of the service")
+
+
+@pytest.mark.timeout(45)
```
Comment (engelmi) [inline tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py]: Nit: I think 15s should be fine.

File: tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        raise Exception(
+                f"{NODE_WITH_NOT_VALID_VALUE} bluechi-agent should "
+                "NOT failed during the start of the service")
+
+    _, output = nodes[NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET].exec_run('systemctl status bluechi-agent')
+    output_systemd = str(output)
+    if "fail" in output_systemd.lower():
+        raise Exception(
+                f"{NODE_WITH_NUMBERS_ONLY_IN_LOGISQUIET} bluechi-agent should "
+                "NOT failed during the start of the service")
```
Comment (engelmi) [inline tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py:42]: Optional:
You could also check if the nodes are shown as on-/offline by using the python bindings (see example [here](https://github.com/containers/bluechi/tree/main/tests/tests/tier0/monitor-open-close)).

**Review by engelmi — APPROVED** [review]
A minor nit, otherwise LGTM

File: tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py (hunk 1 above)
Comment (dougsland) [reply tests/tests/tier0/bluechi-agent-logisquiet/test_agent_logisquiet.py]: Sure, as I have the approval. I will move forward with the patch what we have now just moving to 15 the timeout. The next one I will use bindings, thanks for the review.

**Pull request merged**